tests/testthat/test_mod_filterTab.R

```diff
@@ -0,0 +1,51 @@
+context("Tests for the filterTab R module")
+library(safetyGraphics)
+library(shiny)
+library(shinytest)
+library(testthat)
+library(stringr)
+
+app <- ShinyDriver$new("./module_examples/filterTab")
+initial<-app$getAllValues()
```

Comment:
`getAllValues` returns... calls to `exportTestValues`?

Reply:
It returns a snap shot of all input, output and test values. Full docs here: https://rstudio.github.io/shinytest/reference/ShinyDriver.html#method-getAllValues